pypeit/utils.py

```diff
@@ -163,6 +163,8 @@ def boxcar_smooth_rows(img, nave, wgt=None, mode='nearest'):
     Returns:
```

Comment:
Has this code been tested to reproduce exactly the functionality that was in trace_crude_init?

Reply:
I've added a new test to make sure the function matches what one should get from a brute force approach.  Looking at the code itself, `boxcar_smooth_rows` is identical to the previous lines in `trace_crude_init`, except for the use of the `numpy.ma.divide` function.